config/samples/dataplane_v1beta1_openstackdataplane.yaml

```diff
@@ -36,6 +36,8 @@ spec:
         deploy: false
   roles:
     edpm-compute:
+      services:
+        - configure-network
```

Comment:
In this example only the `configure-network` service is on the services list. If other services are migrated the same way (e.g. `InstallOS`, `ConfigureOS`, etc from deployment.go) then it will be easy for the deployer to introduce deployment break points. 

This would be useful to me for the HCI usecase because I could shorten this list to only `configure-network`, and then once the storage network is configured, I could go and deploy ceph manually. I could then come back and extend the list with `configure-os` etc to finish the deployment.